web/src/components/netflow-traffic.tsx

```diff
@@ -360,6 +343,14 @@ export const NetflowTraffic: React.FC<{
   );
 
   const tick = React.useCallback(() => {
+    // skip tick while forcedFilters & config are not loaded
+    // this check ensure tick will not be called during init
+    // as it's difficult to manage react state changes
+    if (!initState.current.includes('forcedFiltersLoaded') || !initState.current.includes('configLoaded')) {
```

Comment:
I think it would be better to have a union type for `initState` such as array of `'forcedFiltersLoaded' | 'configLoaded' | ...` (e.g. to avoid typos)

Reply:
https://github.com/netobserv/network-observability-console-plugin/pull/273/commits/69b6f1e776c1cf036aa062384fa4211097a86f8d